artifacts_test.py

```diff
@@ -200,6 +200,11 @@ def verify_nvme_write_cache(self) -> None:
             write_cache = run(f"cat /sys/block/{nvme_device.removeprefix('/dev/')}/queue/write_cache").stdout.strip()
             self.assertEqual(write_cache, expected_write_cache_value)
 
+    def check_service_existence(self, service_name):
+        res = self.node.remoter.run(f'systemctl list-units --full | grep -Fq "{service_name}"',
+                                    ignore_status=True)
+        return not res.exit_status
```

Comment:
In my opinion it's better to return boolian

Reply:
It returns a boolian